ports/stm32/eth.c

```diff
@@ -824,6 +923,7 @@ void eth_low_power_mode(eth_t *self, bool enable) {
 
     // Enable eth clock
     #if defined(STM32H7)
+    // TODO: STM32H5
```

Comment:
Can this TODO be resolved?

Reply:
I couldn't test this code path so far. Is there a test case/scenario I can follow to be sure it's actually working as expected?